### PR TITLE
docs: Privileged execution - use nginx instead of test-pod image

### PR DESCRIPTION
**Pull request by janvi01**
Fixes: https://github.com/cilium/tetragon/issues/705
Signed-off-by: janvi01 [janvibajo1@gmail.com](mailto:janvibajo1@gmail.com)

### CODE REVIEW

**Review by tixxdz — APPROVED** [review]
lgtm much appreciated for your contributions!  let's wait for @mtardy review too ;-)

**Comment by jrfastab** [comment]
docs build had some issue with network connections retry fixed it.

**Comment by jrfastab** [comment]
Otherwise thanks for the patch lgtm.

File: docs/content/en/docs/use-cases/process-lifecycle/privileged-execution.md [hunk 1]
```diff
@@ -16,33 +16,66 @@ have. This would help us answer questions like:
...
+
+  ```bash
+  kubectl edit cm -n kube-system tetragon-config
+  ```
+
+* Set the following flags from "false" to "true":
+
+  ```bash
+  # enable-process-cred: true
+  # enable-process-ns: true
```
Comment (mtardy) [inline docs/content/en/docs/use-cases/process-lifecycle/privileged-execution.md:31]: I think we wanted to remove the `#` here.

**Review by mtardy — COMMENTED** [review]
Thanks! I was too late for the review, this is only nit, not a big deal, maybe the `#` in the config is something we would like to fix.

**Pull request merged**